ruby/vampires.rb

```diff
@@ -0,0 +1,136 @@
+# Gathering vampire data with puts statements, and assigning user input
+# to variables
+
+def new_employee
+	puts "What is your name?"
+		name = gets.chomp
+	puts "How old are you?"
+		age = gets.chomp
+		#convert answer to integer
+		age = age.to_i
+	puts "What year were you born?"
+		birth_year = gets.chomp
+		#convert answer to integer
+		birth_year = birth_year.to_i
+	#compare age to birth_year and assign boolean value 
+		if (2016 - birth_year) == age
+		  boolean_age = true
+		 else
+		   boolean_age = false
+		 end
+		p boolean_age
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+		garlic_bread = gets.chomp
+		#convert answer to boolean"
+		if garlic_bread == "yes"
+			garlic_bread = true
+		else
+			garlic_bread = false
+		end
+		p garlic_bread
+	puts "Would you like to enroll in the compnay's health insurance?"
+		insurance = gets.chomp
+		#convert answer to boolean
+		if insurance == "yes"
+			insurance = true
+		else
+			insurance = false
+		end
+		p insurance
+
+#Add loop for allergy question	
+  
+    #loop do
+     # puts "List any allergies you have one at a time, and write done when finished."
+     # allergies_answer = gets.chomp
+     # if allergies_answer == 'done'
+     #   break
+     # end
+     # if allergies_answer == 'sunshine'
+        #allergies_answer = true
+      #else 
+        #allergies_answer = false
+     #   break #break or next? need to skip to result of "probably a vampire"
+     # end  
+    #end 
+    
+    #trying until loop to define variable outside loop 
+    allergies_answer = false
+    answer = ""
+   
+    until allergies_answer
+      puts "List your allergies one at a time, and write done when finished."
+      answer = gets.chomp
+        if answer == 'done'
+          break
+        end  
+        if answer == 'sunshine'
+          break allergies_answer = true
+        end
+    end   
+
+
+	# vampire detection logic added
+	#check all conditions and base result on latest
+	#if/else is returning 1st matching result, not continuing to check,
+	#find another method
+	
+	#while vampire_status != "Results inconclusive" #tried to force code to continue
+	#checking 
+  #	if boolean_age && (garlic_bread || insurance)
+  #	  vampire_status = "Probably not a vampire."
+  #	elsif !boolean_age && (!garlic_bread || !insurance)
+  #	  vampire_status = "Probably a vampire."
+  #	elsif !boolean_age && !garlic_bread && !insurance
+  #	  vampire_status = "Almost certainly a vampire."
+  	#name == "Drake Cula || Tu Fang"
+  	 # vampire_status = "Definitely a vampire."
+   
+  	  #vampire_status = "Results inconclusive"
+  	#end
+  #end
+	 # p vampire_status
+	  
+	#Conditional logic with only if statements 
+	
+	
+	  vampire_status = "Probably not a vampire." if boolean_age && (garlic_bread || insurance) == true
```

Comment:
Generally I like to see chaining `if/elsif/else` but you're making this work and dig it.

You could optimize a little though.  If you have a moment see if you can nest some of these in conditions so we don't run each instance of this.